src/classes/GE_GiftEntryController.cls

```diff
@@ -37,6 +37,31 @@
 */
 public with sharing class GE_GiftEntryController {
 
+    @AuraEnabled(cacheable=true)
+    public static String getOrgDomain() {
+        String url = System.URL.getOrgDomainUrl().getHost();
+        String[] urlParts = url.split('\\.');
+
+        return urlParts[0];
+    }
+
+    @AuraEnabled
+    public static String makePurchaseCall(String token) {
+        GE_PaymentServices paymentServices = new GE_PaymentServices(token);
+        HttpResponse response = paymentServices.makePurchaseCall();
+
+        // Parse the JSON response...
+        if (response.getStatusCode() != 201) {
+            System.debug('The status code returned was not expected: ' +
```

Comment:
Are we going to keep these debug statements for continued dev?  Or should we not commit them... ?

Reply:
I'm gonna go ahead and sweep these out along with console logs/alerts. I was thinking they might be useful as a lot of these methods are just starting points, but I believe our current guidelines are to remove them.